tests/application-registry-tests/test/testkit/k8sresources_check.go

```diff
@@ -73,6 +74,33 @@ func CheckK8sCertificateGenSecret(t *testing.T, secret *v1core.Secret, name stri
 	checkLabels(t, labels, secret.Labels)
 }
 
+func CheckK8sParamsSecret(t *testing.T, secret *v1core.Secret, name string, labels Labels, headerKey, headerValue, queryParameterKey, queryParameterValue string) {
+	require.Equal(t, name, secret.Name)
+
+	const (
```

Comment:
I suggest moving those constants outside of the function.

Reply:
Done!